diff --git a/src/parser/src/RustLexer.ts b/src/parser/src/RustLexer.ts
index 59e1842..de8b9ec 100644
--- a/src/parser/src/RustLexer.ts
+++ b/src/parser/src/RustLexer.ts
@@ -2,6 +2,7 @@
 
 import * as antlr from "antlr4ng";
 import { Token } from "antlr4ng";
+import RustLexerBase from "./RustLexerBase";
 
 
 export class RustLexer extends RustLexerBase {
diff --git a/src/parser/src/RustParser.ts b/src/parser/src/RustParser.ts
index 202b491..393e8a7 100644
--- a/src/parser/src/RustParser.ts
+++ b/src/parser/src/RustParser.ts
@@ -5,6 +5,7 @@ import { Token } from "antlr4ng";
 
 import { RustParserListener } from "./RustParserListener.js";
 import { RustParserVisitor } from "./RustParserVisitor.js";
+import RustParserBase from "./RustParserBase.js";
 
 // for running tests with parameters, TODO: discuss strategy for typed parameters in CI
 // eslint-disable-next-line no-unused-vars
